test/controllers/roles_controller_test.rb

```diff
@@ -51,12 +51,12 @@ class RolesControllerTest < ActionController::TestCase
     assert_nil Role.find_by_id(role.id)
   end
 
-  test 'roles in use cannot be destroyed' do
+  test 'roles in use can be destroyed' do
```

Comment:
Might as well delete this I think, or perhaps change it to a test to check you can't delete built in roles. Your unit test coverage of deletion of roles with users is better I think, and cheaper.